src/components/Question/Question.tsx

```diff
@@ -99,6 +99,18 @@ const Question = (props: QuestionProps): JSX.Element => {
     const isQuantity = props.item.type === IQuestionnaireItemType.quantity;
     const isDecimalOrQuantity = isDecimal || isQuantity;
 
+    // Adds instructions for the user
+    const instructionType = (): JSX.Element => {
+        if(props.item.type === IQuestionnaireItemType.attachment) {
+            return (
+                <p>
+                    <i>Note:</i> ResearchKitonFHIR maps the 'attachment' type to an <strong>ORKImageCaptureStep</strong> which allows the user to take a photo with the device camera.
```

Comment:
Might be nice to add a link to ResearchKitOnFHIR + the "O" should be capitalized: ResearchKit**O**nFHIR.
```suggestion
                    <i>Note:</i> <a href="https://github.com/StanfordBDHG/ResearchKitOnFHIR">ResearchKitOnFHIR</a> maps the 'attachment' type to an <strong>ORKImageCaptureStep</strong> which allows the user to take a photo with the device camera.
```

Reply:
Great suggestion! Thanks @PSchmiedmayer!